site/ja/tutorials/load_data/tf-records.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<blockquote># 整数特徴量&nbsp;-10000 から 10000 の間の乱数</blockquote><blockquote>feature1 = np.random.randint(0, 5, n_observations)</blockquote><p>This comment does not match to the code.</p><p><br></p><blockquote># バイト特徴量</blockquote><blockquote>strings = np.array([b'cat', b'dog', b'chicken', b'horse', b'goat'])</blockquote><p>"文字列特徴量" or "バイト文字列特徴量" may be better than "バイト特徴量" since the original text is "string feature". </p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>